container/src/main/java/com/flipkart/poseidon/api/JettyFilterConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package com.flipkart.poseidon.api;
+
+import org.eclipse.jetty.servlet.FilterMapping;
+
+import javax.servlet.DispatcherType;
+import javax.servlet.Filter;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by shrey.garg on 21/05/16.
+ */
+public class JettyFilterConfiguration {
+    private Filter filter;
+    private List<String> mappings;
+    private Map<String, String> initParameters = new HashMap<>();
+    private EnumSet<DispatcherType> dispatcherTypes;
+
+    public JettyFilterConfiguration(Filter filter, List<String> mappings, EnumSet<DispatcherType> dispatcherTypes) {
```

Comment:
Make EnumSet and mappings optional (provide setters). Provide default as /* and EnumSet.of(REQUEST)